clients/vscode/package.json

```
@@ -194,7 +205,8 @@
       ],
       "editor/context": [
         {
-          "submenu": "tabby.submenu"
+          "submenu": "tabby.submenu",
+          "group": "2_workspace"
```

I think we should use separate group
```suggestion
          "group": "2_tabby"
```